sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/StringOrIntegralTypesForIndexers.CSharp9.cs

```diff
@@ -0,0 +1,18 @@
+﻿record Foo
+{
+    public int this[Class index] { get { return 0; } }
+//                  ^^^^^ Noncompliant {{Use string or an integral type here, or refactor this indexer into a method.}}
+
+    public int this[Record index] { get { return 0; } }
```

Comment:
should add // FN ?

Reply:
to avoid another round, it's ok to just mark it in the spread sheet that we implement it